cli/node/mod.rs

```diff
@@ -668,8 +671,9 @@ fn package_config_resolve(
 ) -> Result<Option<PathBuf>, AnyError> {
   let package_json_path = package_dir.join("package.json");
   let referrer = ModuleSpecifier::from_directory_path(package_dir).unwrap();
+  let permissions = &mut PermissionsContainer::allow_all();
```

Comment:
I'm not sure if this is correct permission here - maybe it should be passed from the caller?

Reply:
We already have such a method - `PackageJson::load_skip_read_permission` but it seems functions from this file can be called from dynamic imports which should check permissions

Reply:
Ok, maybe just to be safe we should pass it from the caller then? I'm not sure though.